contributing/data-model-schema.txt

```diff
@@ -61,9 +61,8 @@ Create the new schema directory
   the schema directory for the current release, so skip over this part.
 
 For the schema release process a high fraction of the necessary work
-occurs in Bio-Formats' :bf_source:`components/specification` directory.
-Inside there, :bf_source:`released-schema
-<components/specification/released-schema>` contains a subdirectory for
+occurs in Bio-Formats' `components/specification` directory.
```

Comment:
Single backticks are (always) invalid in sphinx.  It needs to be double-backticks or `:file:` to mark it up correctly.